src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2854,6 +2854,27 @@ public abstract partial class CriticalFinalizerObject
         protected CriticalFinalizerObject() { }
         ~CriticalFinalizerObject() { }
     }
+    
+    public enum Cer
+    {
+        MayFail = 1,
+        None = 0,
+        Success = 2,
+    }
+    public enum Consistency
+    {
+        MayCorruptAppDomain = 1,
+        MayCorruptInstance = 2,
+        MayCorruptProcess = 0,
+        WillNotCorruptState = 3,
+    }
+    [System.AttributeUsageAttribute((System.AttributeTargets)(1133), Inherited=false)]
+    internal sealed partial class ReliabilityContractAttribute : System.Attribute
```

Comment:
Will change it once the coreclr changes get through https://github.com/dotnet/coreclr/pull/7328